database/index.js

```diff
@@ -50,4 +44,5 @@ let find = (listing, callback) => {
 
 
 module.exports.save = save;
-module.exports.find = find;
\ No newline at end of file
+module.exports.find = find;
+module.exports.review = Review;
```

Comment:
You may want to have exports.Review (capital R) so that the naming is consistent, and to follow the model naming pattern